_pages/faq.md

```diff
@@ -61,6 +61,29 @@ Yes! As long as multiple repositories represent one project and project maintain
 
 Open Source Day does not give contributors a shortcut to their contributions. All participants will be required to abide by the project's code of conduct and other guidelines. The Open Source Day team does not expect any of our project partners to accept/merge any work that does not meet the project’s standards and expectations. We kindly ask you to bear in mind that some participants may be new to open source as you decide how to handle the contributions made to your project.
 
+### Can we have extra materials to guide participants that are not necessarily on Github?
+
+Yes, it is acceptable to have other contribution guides to make it easier for those wishing to contribute to your project. Kindly make sure to let the contributors know before they start their contributions.
+
+### Is it possible to know the number of registered participants to our projects before OSD?
+
+The number of registered participants for each project will be sent to the project representatives to enable them to prepare prior to OSD. However, this might be sent at a closer date to OSD.
+
+### Will we be merging PRs only as project representatives on OSD?
```

Comment:
Hi @phyleria I'm not quite sure what this question means. Is this question in regards to the responsibility of project representatives during the day?